apitest/src/main/resources/partner/policy/GetPolicyGroup/GetPolicyGroup.yml

```diff
@@ -1,12 +1,13 @@
 getPolicyGroup:
    Partner_GetPolicyGroup_All_Valid_Smoke:
       endPoint: /v1/policymanager/policies/group/{policyGroupId}
+      description: Retrieve Parter Policy Group
       role: partner
       restMethod: get
       inputTemplate: partner/policy/GetPolicyGroup/getPolicyGroup
       outputTemplate: partner/policy/GetPolicyGroup/getPolicyGroupResult
       input: '{
-	"policyGroupId": "$ID:DefinePolicyGroup_All_Valid_Smoke_sid_id$"
```

Comment:
Undo the changes done other than description addition

Reply:
Updated changes